homeassistant/components/mobile_app/logbook.py

```diff
@@ -0,0 +1,59 @@
+"""Describe mobile_app logbook events."""
+from __future__ import annotations
+
+from collections.abc import Callable
+
+from homeassistant.const import ATTR_FRIENDLY_NAME, ATTR_ICON
+from homeassistant.core import Event, HomeAssistant, callback
+
+from .const import DOMAIN
+
+IOS_EVENT_FINISHED_LAUNCHING = "ios.finished_launching"
+IOS_EVENT_ENTERED_BACKGROUND = "ios.entered_background"
+IOS_EVENT_BECAME_ACTIVE = "ios.became_active"
+IOS_EVENT_NOTIFICATION_ACTION_FIRED = "ios.notification_action_fired"
```

Comment:
this is the older event, which we've replaced with `mobile_app_notification_action`

Reply:
Do we need to still fire it?

Reply:
I've been avoiding making backwards-incompatible changes as much as possible, so probably: no, but also sorta yes

Reply:
Let us not make them them more obvious that they exist since the goal is to make them go away eventually.  Sadly they take up space in the database.

I'm going to close this PR.

Reply:
just to clarify: the enter/exit events and such are all going to continue firing, just the notification_action_fired is the one we don't mention in docs or anything

Reply:
Oh I misunderstood

I old added descriptions for the enter/exit events since I didn't think the other ones were useful for the logbook anyways!

```python3
    async_describe_event(DOMAIN, IOS_EVENT_ZONE_ENTERED, async_describe_zone_event)
    async_describe_event(DOMAIN, IOS_EVENT_ZONE_EXITED, async_describe_zone_event)
```

Reply:
I removed the constants for the other event types.